recipes/custodian/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "custodian" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "30bb40a3eaa7dbbbfe01de74434d38ad4cdf4b5898b123b00631e9dcb194b5fa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+      - cstdn = custodian.cli.cstdn:main
+      - run_vasp = custodian.cli.run_vasp:main
+      - run_nwchem = custodian.cli.run_nwchem:main
+      - converge_kpoints = custodian.cli.converge_kpoints:main
+      - converge_geometry = custodian.cli.converge_geometry:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - monty
+    - six
+    - pyyaml
+
+test:
+  imports:
+    - custodian
+    - custodian.test_custodian
```

Comment:
This import looks to be invalid. Please have, but it looks like it is in `custodian.tests.test_custodian`. That said, typically you don't have to import the tests module.